multipaz/src/androidMain/kotlin/org/multipaz/util/Logger.android.kt

```diff
@@ -0,0 +1,21 @@
+package org.multipaz.util
+
+import android.util.Log
+import org.multipaz.util.Logger.Level
+
+internal actual fun platformLogPrinter(level: Logger.Level, tag: String, msg: String, throwable: Throwable?) {
+    // Android clamps the log message at just over 4000 characters so chunk and only include
+    // the throwable with the first Log.x() call
+    val messages = msg.chunked(4000)
+    var t = throwable
+    for (chunk in messages) {
+        when (level) {
+            Level.DEBUG -> t?.let { Log.d(tag, chunk, it) } ?: Log.d(tag, chunk)
+            Level.INFO -> t?.let { Log.i(tag, chunk, it) } ?: Log.i(tag, chunk)
+            Level.WARNING -> t?.let { Log.w(tag, chunk, it) } ?: Log.w(tag, chunk)
+            Level.ERROR -> t?.let { Log.e(tag, chunk, it) } ?: Log.e(tag, chunk)
+            else -> throw IllegalArgumentException("Unknown log level: $level")
```

Comment:
Now that Level is an enum class instead of an int, do we need the else?

Reply:
We do not. Fixed.